### PR TITLE
Feat/contracts list

### PR DESCRIPTION
**Pull request by franzspiess**

### CODE REVIEW

File: client/src/components/ContractList/ContractList.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    return (
+      <div key={z} className="my-contract">
+        <i className="fas fa-file-contract" />
+        <span className="contract-title">{contract.title}</span>
+      </div>
+    )
+  })
+
+
+
```
Comment (gaber85) [inline client/src/components/ContractList/ContractList.js]: nit: unnecessary spaces

**Review by gaber85 — CHANGES_REQUESTED** [review]
spacing and removing extra files

File: client/src/App.js [hunk 2]
```diff
@@ -1,14 +1,30 @@
...
-import SignUp from './components/SignUp';
+// import SignUp from './components/SignUp';
 // import NavBar from './components/NavBar';
+import ContractList from './components/ContractList'
 
 // eslint-disable-next-line
 class App extends Component {
   render() {
-    return <SignUp/>;
+    return <ContractList />;
```
Comment (oliversisson) [inline client/src/App.js]: Are you sure you want to check this bit?

File: client/src/components/ContractList/ContractList.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+const ContractList = () => {
+
+  let z = -1;
+
+
+  const contracts =
+    [{ title: 'My Contract' }, { title: 'My Contract' }, { title: 'My Contract' }, { title: 'My Contract' }, { title: 'My Contract' }, { title: 'My Contract' }, { title: 'My Contract' }, { title: 'My Contract' }, { title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' }]
+
+
+
```
Comment (oliversisson) [inline client/src/components/ContractList/ContractList.js]: nit: extra lines

File: client/src/components/ContractList/ContractList.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  const view = contracts.map(contract => {
+    z += 1;
```
Comment (oliversisson) [inline client/src/components/ContractList/ContractList.js]: delete "z" and use this instead:
```
contracts.map((contract, index) => {
...
}
```
https://stackoverflow.com/questions/38364400/index-inside-map-function

Comment (oliversisson) [reply client/src/components/ContractList/ContractList.js]: also, if it's an index, why not call it `i` or `index`?

File: client/src/components/ContractList/ContractList.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    return (
+      <div key={z} className="my-contract">
+        <i className="fas fa-file-contract" />
```
Comment (oliversisson) [inline client/src/components/ContractList/ContractList.js:17]: https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

File: client/src/components/ContractList/ContractList.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    )
+  })
+
+
+
+
+
+  return (
+    <div className="contract-list">
+      {view}
```
Comment (oliversisson) [inline client/src/components/ContractList/ContractList.js:25]: inline `view`

File: client/src/components/PlusButton/PlusButton.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+    padding: `${size*0.1}vw`
+  }
+
+  return (
+    <div className="plus-button" style={myStyle} tabIndex="0" role="button" onClick={banana} onKeyPress={banana}><i className="fas fa-plus" /></div>
+  )
+
+}
+
+export default PlusButton;
```
Comment (oliversisson) [inline client/src/components/PlusButton/PlusButton.js:20]: nit: extra lines

File: client/src/components/PlusButton/PlusButton.js [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+  const myStyle = {
+    width: `${size}vw`,
+    height: `${size}vw`,
+    fontSize: `${size}vw`,
+    padding: `${size*0.1}vw`
+  }
+
+  return (
+    <div className="plus-button" style={myStyle} tabIndex="0" role="button" onClick={banana} onKeyPress={banana}><i className="fas fa-plus" /></div>
```
Comment (oliversisson) [inline client/src/components/PlusButton/PlusButton.js]: nit: split lines

is `banana` really the best name?

**Review by oliversisson — COMMENTED** [review]
lgtm, have a look at my comments

**Review by gaber85 — APPROVED** [review]

**Pull request merged**